composer/utils/object_store.py

```diff
@@ -286,7 +288,36 @@ def upload_object_via_stream(self,
                                                 headers=headers)
 
     def _get_object(self, object_name: str):
-        return self._provider.get_object(self._container.name, object_name)
+        """Get object from object store. Recursively follow any symlinks. If an object does not exist, automatically
+        check if it is a symlink by appending ``.symlink``.
+
+        Args:
+            object_name (str): The name of the object.
+        """
+        obj = None
+        try:
+            obj = self._provider.get_object(self._container.name, object_name)
+        except Exception as e:
```

Comment:
At one point in time, `libcloud` was an optional dependency, which is why we did the conditional comparison. But now it is included in core.

```suggestion
        except ObjectDoesNotExistError:
```